DependencyInjection/Configuration.php

```diff
@@ -43,18 +43,29 @@ public function getConfigTreeBuilder()
 
                 ->arrayNode('persistence')
                     ->addDefaultsIfNotSet()
+                    ->beforeNormalization()
+                        ->ifTrue(function ($v) {
+                            return !isset($v['service']) && !isset($v['class']);
+                        })
+                        ->then(function ($v) {
+                            $v['class'] = 'Liip\TranslationBundle\Persistence\YamlFilePersistence';
+                            $v['options']['folder'] = '%kernel.root_dir%/data/translations';
```

Comment:
can't you leave this below as default value? i think right now you will always overwrite the value when only the folder was configured but not the class...